tests/test_languages.py

```diff
@@ -10,7 +10,6 @@
 
 from tests import BaseTestCase
 
-
```

Comment:
double blank lines are recommended between imports and class definition. Please revert this change.